### PR TITLE
Bootstrap 5.3 and dark mode

### LINKED ISSUE #4450: Bootstrap 5.3 and dark mode for docs
Bootstrap 5.3 alpha 1 was release https://blog.getbootstrap.com/2022/12/24/bootstrap-5-3-0-alpha1/ so it will be nice to have a doc with dark mode to test the components.

[EDIT] Bootstrap 5.3 is out of alpha now https://blog.getbootstrap.com/2023/05/30/bootstrap-5-3-0/

### PR DESCRIPTION
**Pull request by fbasso**
Rebase of #4451 + some other fixes

Fix #4450

### CODE REVIEW

File: demo/src/app/shared/component-wrapper/component-wrapper.component.html
Comment (bastienmoulia) [inline demo/src/app/shared/component-wrapper/component-wrapper.component.html:1]: You can replace `style="border-right: 1px solid rgba(0, 0, 0, 0.1)"` by the class `border-end` to show the border in dark mode.

File: demo/src/app/shared/page-wrapper/page-wrapper.component.html
Comment (bastienmoulia) [inline demo/src/app/shared/page-wrapper/page-wrapper.component.html:1]: You can replace `style="border-right: 1px solid rgba(0, 0, 0, 0.1)"` by the class `border-end` to show the border in dark mode.

File: demo/src/style/app.scss
Comment (bastienmoulia) [inline demo/src/style/app.scss:1]: You can replace `color: rgba(0, 0, 0, 0.85);` by `color: rgba(var(--bs-emphasis-color-rgb), 0.85);`

**Review by bastienmoulia — CHANGES_REQUESTED** [review]
Thanks for taking this issue, I have some small suggestions to improve the documentation in dark mode.

**Comment by fbasso** [comment]
Thanks for your help @bastienmoulia !

All your suggestions are valid, I have integrated them. The PR should be ready to be merge soon I think.

**Comment by bastienmoulia** [comment]
Do you need some help to finish it?

**Pull request merged**